packages/api/models/rules.js

```diff
@@ -254,7 +254,7 @@ class Rule extends Manager {
       newRuleItem = await this.validateAndUpdateSqsRule(newRuleItem);
       break;
     default:
-      throw new Error('Type not supported');
+      throw new Error(`Rule type ${newRuleItem.rule.type} not supported.`);
```

Comment:
My only suggestion would be to put the type quotes (single or double) within the message to avoid having it look like a regular word in the message. For example: `Rule type "foo" not supported`. This also covers the empty string, which is not noticeable without quotes.

Reply:
@chuckwondo I made these updates.